### DISCUSSION TITLE
Changes are not made

### DISCUSSION THREAD

**Discussion by AndresVargasA** [cyberbotics/wrestling#68] [Q&A] [answered]
Our robot Kaizen not change when we do push, please can tell us why the changes not working? Thanks

**Comment by MJavadZallaghi** [comment]
So do you receive failure in the run by server?
I have the same problem.

**Comment by omichel** [comment, marked as answer]
It seems the simulation server was messed-up with some system update. It is now fixed. You should be able to re-run your games now.

**Reply by AndresVargasA** [reply]
we still have the problem, we do push and no changes are made on the server so the robot does not perform any fight.

**Reply by omichel** [reply]
It seems this is because you deleted your local .github folder which contained the workflows necessary to trigger the evaluation.

**Reply by omichel** [reply]
I just fixed it on your local repository, see https://github.com/danielebg/KAIZEN/commit/6c18be722eb3af4de4e4817b8ecdb74bcccedd1c. It should work again now.

**Reply by omichel** [reply]
You should also probably remove this line: https://github.com/danielebg/KAIZEN/blob/main/controllers/participant/participant.py#L10 which is preventing your controller to connect to the simulation...

**Reply by AndresVargasA** [reply]
now working, thank you
